src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Stack;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Stack<Ball> ballStack = new Stack<>();
+
+        for (int i = 0; i < 3; i++) {
+            ballStack.push(lottery.getRandomBall());
+        }
+
+        while (!ballStack.isEmpty()) {
+            System.out.println(ballStack.pop());
+        }
```

Comment:
By `write in stack` previous mentor meant that you have to write each enum value on a new line. Please revert your previous main method implementation